test/Microsoft.ML.StaticPipelineTesting/TreeRepresentation.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Microsoft.ML.Data;
+using Microsoft.ML.RunTests;
+using Microsoft.ML.StaticPipe;
+using Microsoft.ML.Trainers.FastTree;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.ML.StaticPipelineTesting
+{
+    public sealed class TreeRepresentation : BaseTestClassWithConsole
+    {
+        public TreeRepresentation(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public void FastTreeRegressionRepresentation()
+        {
+            var env = new MLContext(seed: 0);
+            var dataPath = GetDataPath(TestDatasets.generatedRegressionDataset.trainFilename);
+            var dataSource = new MultiFileSource(dataPath);
+
+            var catalog = new RegressionCatalog(env);
+
+            var reader = TextLoaderStatic.CreateReader(env,
+                c => (label: c.LoadFloat(11), features: c.LoadFloat(0, 10)),
+                separator: ';', hasHeader: true);
+
+            FastTreeRegressionModelParameters pred = null;
+
+            var est = reader.MakeNewEstimator()
+                .Append(r => (r.label, score: catalog.Trainers.FastTree(r.label, r.features,
+                    numTrees: 10,
+                    numLeaves: 5,
+                    onFit: (p) => { pred = p; })));
+
+            var pipe = reader.Append(est);
+
+            Assert.Null(pred);
+            var model = pipe.Fit(dataSource);
+            Assert.NotNull(pred);
+
+            var treeCollection = pred.TrainedTreeCollection;
+            Assert.Equal(0, treeCollection.Bias);
+            Assert.Equal(10, treeCollection.Trees.Count);
+            Assert.Equal(10, treeCollection.TreeWeights.Count);
+
+            var trees = treeCollection.Trees;
```

Comment:
>trees [](start = 16, length = 5)

Please check the number of trees as well.

Reply:
Checked above at line 45.

---
In reply to: [252029977](https://github.com/dotnet/machinelearning/pull/2243#discussion_r252029977) [](ancestors = 252029977)

Reply:
ok.

---
In reply to: [252030449](https://github.com/dotnet/machinelearning/pull/2243#discussion_r252030449) [](ancestors = 252030449,252029977)